### PR TITLE
#1109 Improve pit test coverage

### PR DESCRIPTION
**Pull request by goldbal330**
Signed-off-by: Alexander Goldberg <alexander.goldberg@bnymellon.com>

### CODE REVIEW

**Comment by goldbal330** [comment]
PR submitted

File: unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/AdaptObjectIntProcedureToProcedureTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021 Goldman Sachs.
```
Comment (donraab) [inline unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/AdaptObjectIntProcedureToProcedureTest.java:2]: Add "and others" after GS.

File: unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/CaseProcedureTest.java [hunk 2]
```diff
@@ -29,7 +29,7 @@ public void noopCaseAndThenDefault()
         CaseProcedure<String> procedure = new CaseProcedure<>();
         strings.each(procedure);
         Verify.assertEmpty(result);
-        procedure.setDefault(result::add);
+        Verify.assertSame(procedure, procedure.setDefault(result::add));
```
Comment (donraab) [inline unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/CaseProcedureTest.java:32]: Add "and others" in the copyright.

File: unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/ChainedProcedureTest.java [hunk 3]
```diff
@@ -35,4 +36,13 @@ public void procedure()
         Assert.assertEquals(list, list1);
         Assert.assertEquals(list, list2);
     }
+
```
Comment (donraab) [inline unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/ChainedProcedureTest.java:39]: Same comment on copyright.

File: unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/IfProcedureTest.java [hunk 4]
```diff
@@ -16,6 +16,7 @@
 import org.eclipse.collections.impl.list.mutable.FastList;
 import org.eclipse.collections.impl.test.Verify;
 import org.eclipse.collections.impl.utility.Iterate;
+import org.eclipse.collections.impl.utility.StringIterate;
```
Comment (donraab) [inline unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/IfProcedureTest.java:19]: Same comment on copyright.

**Review by donraab — CHANGES_REQUESTED** [review]
In progress reviewing. Please address consistency issues on the copyright notices. Thanks!

File: unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/CollectionAddProcedureTest.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+package org.eclipse.collections.impl.block.procedure;
+
+import org.eclipse.collections.api.factory.Lists;
+import org.eclipse.collections.impl.utility.StringIterate;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CollectionAddProcedureTest
```
Comment (prathasirisha) [inline unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/CollectionAddProcedureTest.java:18]: Hi Alex, noticed there were few other methods in this class that are not being covered in the test cases. Can you add tests for those as well?

**Pull request closed without merging**